client/src/components/user-admin/UserManagement.js

```diff
@@ -42,69 +49,108 @@ const UserManagement = ({ users, setUserToEdit }) => {
             );
   }
   return (
-    <div className="container--usermanagement">
-      <div>
-        <h3>User Management</h3>
-        <input
+    <Box className="container--usermanagement" sx={{px: '1.8rem'}}>
+      <Box sx={{
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        pt: '40px',
```

Comment:
Please change this to be a numerical value via MUI's spacing system. I believe the equivelent to `pt: '40px',` is `pt: 5` in our current MUI theme configuration.